components/brave_wallet/common/string_utils_unittest.cc

```diff
@@ -60,47 +60,48 @@ TEST(StringUtilsUnitTest, Base10ValueToUint256) {
 TEST(StringUtilsUnitTest, Base10ValueToInt256) {
   int256_t out;
   ASSERT_TRUE(Base10ValueToInt256("0", &out));
-  ASSERT_EQ(out, (int256_t)0);
+  EXPECT_EQ(out, int256_t(0));
   ASSERT_TRUE(Base10ValueToInt256("1", &out));
-  ASSERT_EQ(out, (int256_t)1);
+  EXPECT_EQ(out, int256_t(1));
   ASSERT_TRUE(Base10ValueToInt256("-1", &out));
-  ASSERT_EQ(out, (int256_t)-1);
+  EXPECT_EQ(out, int256_t(-1));
   ASSERT_TRUE(Base10ValueToInt256("12345678910", &out));
-  ASSERT_EQ(out, (int256_t)12345678910);
+  EXPECT_EQ(out, int256_t(12345678910));
   ASSERT_TRUE(Base10ValueToInt256("-12345678910", &out));
-  ASSERT_EQ(out, (int256_t)-12345678910);
+  EXPECT_EQ(out, int256_t(-12345678910));
 
   // Max int256 value can be represented
-  int256_t expected_val = std::numeric_limits<int256_t>::max();
+  int256_t expected_val = kMax256BitInt;
   ASSERT_TRUE(
       Base10ValueToInt256("5789604461865809771178549250434395392663499233282028"
                           "2019728792003956564819967",
                           &out));
-  ASSERT_TRUE(out == (int256_t)expected_val);
+  EXPECT_EQ(out, expected_val);
 
   // Min int256 value can be represented
-  expected_val = std::numeric_limits<int256_t>::min();
-  ASSERT_TRUE(Base10ValueToInt256(
-      "-5789604461865809771178549250434395392663499233282028"
-      "2019728792003956564819968",
-      &out));
-  ASSERT_TRUE(out == (int256_t)expected_val);
+  expected_val = kMin256BitInt;
+  ASSERT_TRUE(
+      Base10ValueToInt256("-578960446186580977117854925043439539266349923328202"
```

Comment:
why did these change?

Reply:
@darkdh ping :point_up:

Reply:
I'm fixing it, one char is stripped by accident

Reply:
It's actually same value just get cut off at different location
“-5789604461865809771178549250434395392663499233282028” “2019728792003956564819968"
“-578960446186580977117854925043439539266349923328202” “82019728792003956564819968"
I will make it like original value

Reply:
fixed iin 83ee7c03da67f29b4b26ef8ea069077f8ec6b72e